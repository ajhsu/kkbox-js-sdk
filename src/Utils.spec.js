const { createAuthorizationHeader } = require('./Utils');

describe('Utils', () => {
  test('createAuthorizationHeader', () => {
    // Use-cases
    expect(createAuthorizationHeader('fakeAccessToken')).toMatchSnapshot();
  });
});
