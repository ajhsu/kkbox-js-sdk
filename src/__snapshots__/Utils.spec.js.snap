// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils createAuthorizationHeader 1`] = `
Object {
  "headers": Object {
    "authorization": "Bearer fakeAccessToken",
  },
}
`;
